README.md

```diff
@@ -20,9 +20,7 @@ bundle install
 touch .env
 
 # Clone production data
-rake db:create db:migrate
-rake cascade:sync
-rake faculty:sync
+cap production db:clone
```

Comment:
If this follows an established pattern, then I'm more amenable to it. But I don't see any similar commands for inside, at least in the README.